be/src/storage/lake/tablet_manager.cpp

```diff
@@ -23,6 +23,7 @@
 #include "agent/master_info.h"
 #include "common/compiler_util.h"
 #include "common/config.h"
+#include "exec/schema_scanner/schema_be_tablets_scanner.h"
```

Comment:
will it be ok to introduce this dependency from storage? it will be a bi-direction dependency, storage refers something from exec/schema_scanner, and exec/scanner refers something from storage. maybe extract the common structure definition outside the schema_be_tablets_scanner.h, will be better?